### PR TITLE
Pull Request Airport Challenge 

### PR DESCRIPTION
**Pull request by kate102**
The current status of this challenge is the following:

------------------------------------------------------------------------
The results of rspec
Finished in 0.00388 seconds (files took 0.19891 seconds to load)
12 examples, 0 failures
COVERAGE: 100.00% -- 83/83 lines in 6 files

------------------------------------------------------------------------
The results of rubocop
Inspecting 9 files
.........
9 files inspected, no offenses detected

------------------------------------------------------------------------
I got to the stage where I needed to write a stub to override random weather conditions.
I ran out of time looking up stubs.

To Do:
Test stub
Edge Cases
Doubles

### CODE REVIEW

File: lib/weather.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# This is the weather class
+
+class Weather
+ 
+  attr_writer :condition
```
Comment (Brendao1) [inline lib/weather.rb]: Could this be changed to attr_accessor?
